pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -150,7 +150,9 @@ func (dswp *desiredStateOfWorldPopulator) Run(sourcesReady config.SourcesReady,
 
 func (dswp *desiredStateOfWorldPopulator) ReprocessPod(
```

Comment:
The comment on this function is out of date now.

Reply:
How was this fixed. The comment still reads:

> // ReprocessPod removes the specified pod from the list of processedPods
	// (if it exists) forcing it to be reprocessed. This is required to enable
	// remounting volumes on pod updates (volumes like Downward API volumes
	// depend on this behavior to ensure volume content is updated).

Reply:
Updated the comment.